packages/core/src/graphql/index.ts

```diff
@@ -0,0 +1,72 @@
+import { graphiQLHtml } from "@/ui/graphiql.html.js";
+import { maxAliasesPlugin } from "@escape.tech/graphql-armor-max-aliases";
+import { maxDepthPlugin } from "@escape.tech/graphql-armor-max-depth";
+import { maxTokensPlugin } from "@escape.tech/graphql-armor-max-tokens";
+import { createYoga } from "graphql-yoga";
+import { createMiddleware } from "hono/factory";
+import { buildGraphQLSchema } from "./buildGraphqlSchema.js";
+import { buildLoaderCache } from "./buildLoaderCache.js";
+
+/**
+ * Middleware for GraphQL with an interactive web view.
+ *
+ * - Docs: [TODO(kyle)]
+ *
+ * @example
+ * import { ponder } from "@/generated";
+ * ponder.use("/graphql", graphql());
+ *
+ */
+export const graphql = (
+  {
+    maxOperationTokens = 1000,
+    maxOperationDepth = 100,
+    maxOperationAliases = 30,
+  }: {
+    maxOperationTokens?: number;
+    maxOperationDepth?: number;
+    maxOperationAliases?: number;
+  } = {
+    // Default limits are from Apollo:
+    // https://www.apollographql.com/blog/prevent-graph-misuse-with-operation-size-and-complexity-limit
+    maxOperationTokens: 1000,
+    maxOperationDepth: 100,
+    maxOperationAliases: 30,
+  },
+) =>
+  createMiddleware(async (c) => {
+    const readonlyStore = c.get("readonlyStore");
```

Comment:
Could be interesting to attempt to implement the GraphQL resolvers using the drizzle select API.

But in the meantime, we might want to obfuscate this key name if we're not going to expose/document it. Same for metadataStore.